controller/lib/core/sensors.h

```diff
@@ -86,8 +104,28 @@ class Sensors {
   // Calibrated average sensor values in a zero state.
   Voltage sensors_zero_vals_[NUM_SENSORS];
 
-  // State related to integrating volume from flow.
+  // Our flow sensors are subject to some zero-point drift, meaning that the
+  // flow we observe is equal to true flow plus an error, and that error
+  // changes over time.
```

Comment:
Comment that the error has both a high-frequency (noise) and a low-frequency (drift) component and that PID takes care of the low-frequency one, while high-frequency is addressed I guess by repeatedly reading sensors several times?